mleap-core/src/main/scala/ml/combust/mleap/core/classification/LogisticRegressionModel.scala

```diff
@@ -46,22 +46,6 @@ case class ProbabilisticLogisticsRegressionModel(coefficientMatrix: Matrix,
   override val numClasses: Int = interceptVector.size
   override val numFeatures: Int = coefficientMatrix.numCols
 
-  lazy val binaryModel: BinaryLogisticRegressionModel = {
```

Comment:
@hollinwilkins this doesn't seem to be used anywhere, I decided to remove it.

Reply:
Probably unneeded. I may have just taken it from Spark.